package.json

```diff
@@ -8,8 +8,9 @@
     "start": "next start"
   },
   "dependencies": {
-    "next": "^9.4.4",
+    "next": "^9.5.1",
     "react": "^16.13.1",
-    "react-dom": "^16.13.1"
+    "react-dom": "^16.13.1",
+    "react-transition-group": "^4.4.1"
   }
 }
```

Comment:
Can you delete this file? We're using `yarn.lock` as the source of truth for dependencies. So if you want to test things locally and want to make sure it matches up with what's deployed, use `yarn install` to install the dependencies.

Reply:
oh wait nvm

Reply:
hahaha I thought this was a `package-lock.json`